tool/jt.rb

```diff
@@ -340,11 +333,22 @@ def replace_env_vars(string, env = ENV)
   end
 
   def sh(*args)
-    Dir.chdir(TRUFFLERUBY_DIR) do
+    chdir(TRUFFLERUBY_DIR) do
       raw_sh(*args)
     end
   end
 
+  def chdir(dir, &block)
+    raise LocalJumpError unless block_given?
```

Comment:
this might be confusing without any message. Argument error might be better fit.

Reply:
It's the typical error on missing block, like `yield` generates. I'll add a message.

Reply:
Fixed.

Reply:
yeah but methods customary check explicitly for the block and raise something more friendly